src/rectangle.ts

```diff
@@ -150,9 +150,8 @@ export class Rectangle implements GeoShape {
     return this.shift(p.x, p.y);
   }
 
-  equals(_other: Polygon) {
-    // TODO Implement
-    return false;
+  equals(other: Polygon) {
+    return Polygon.collision(this.polygon, other);
```

Comment:
I don't think this is correct – we should actually check that the rectangles are equal